src/org/labkey/test/tests/SampleTypeNameExpressionTest.java

```diff
@@ -232,4 +286,298 @@ private void verifyNames(String sampleTypeName, String header, String nameExpres
         assertEquals("Fourth name not as expected", name1, names.get(3));
     }
 
+    /**
+     * <p>
+     *     Verify that a derived sample, using a name expression, can be created through the UI.
+     * </p>
+     * <p>
+     *     This test will:
+     *     <ul>
+     *         <li>Create a derived sample using the UI and the name expression to name it.</li>
+     *     </ul>
+     * </p>
+     * @throws Exception Can be thrown by test helper.
+     */
+    @Test
+    public void testCreateDerivedThroughUI() throws Exception
```

Comment:
I updated the function name to `testDeriveSampleFromSampleDetailsPage`.
As far as I can tell all the other tests in this file use import to create derived samples, this is the only one (in this test class) that uses the UI. Because of that I wanted to make sure the test name conveys that info.
I have added clarify text to the issue comments.